palace.pyx

```diff
@@ -1158,6 +1160,97 @@ cdef class SourceGroup:
         """
         self.impl.destroy()
 
+
+cdef class AuxiliaryEffectSlot:
+    """An effect processor.
+
+    It takes the output mix of zero or more sources,
+    applies DSP for the desired effect (as configured
+    by a given Effect object), then adds to the output mix.
+
+    Parameters
+    ----------
+    context : Optional[Context]
+        The context from which the auxiliary effect slot is to be created.
+        If it is `None`, the object is left uninitialized.
+    """
+    cdef alure.AuxiliaryEffectSlot impl
+
+    def __init__(self, context: Optional[Context]) -> None:
+        if context is None: return
+        self.impl = (<Context> context).impl.create_auxiliary_effect_slot()
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exc_type: Optional[Type[BaseException]],
+                 exc_val: Optional[BaseException],
+                 exc_tb: Optional[TracebackType]) -> Optional[bool]:
+        self.destroy()
```

Comment:
The context manager should be briefly mentioned, e.g. like in `Device`.

Reply:
I'm not sure what you mean by that. It's exactly the same, except that in `Device`, `self.close()` is called instead.

Reply:
I mean to be mentioned in the class docstring.

Reply:
Gentle ping.